app/views/reports/_total_inventory_value_table.html.erb

```diff
@@ -1,24 +1,33 @@
 <table class="table table-striped table-responsive data-table">
   <thead>
     <tr>
-      <th class="sort-asc">Description</th>
-      <th class="text-center monetary-value">Value</th>
+      <th class="sort-asc">Category</th>
+      <th>Item</th>
+      <th class="text-center num-value">Count</th>
+      <th class="text-center monetary-value-1">Value</th>
+      <th class="text-center monetary-value-2">PPV</th>
     </tr>
   </thead>
 
   <tbody>
-    <% report.each do |description, value| %>
+    <% report.each do |category_name, item_name, count, value, ppv| %>
       <tr>
-        <td><%= description %></td>
+        <td><%= category_name %></td>
+        <td><%= item_name %></td>
+        <td class="text-center"><%= count %></td>
         <td class="text-center"><%= number_to_currency value, precision: 2 %></td>
+        <td class="text-center"><%= number_to_currency ppv, precision: 2 %></td>
       </tr>
     <% end %>
   </tbody>
 
   <tfoot>
     <tr>
       <th></th>
-      <th class="text-center"><%= number_to_currency report.total_value, precision: 2 %></th>
```

Comment:
Question for @LisaJBlanchard : Do you want the total at the bottom of these reports to show the total for all pages of the report, or just the current page being viewed? If it should be the total for just this page of data, should there be another place that includes the full total?